mlir/include/mlir/Dialect/Tosa/IR/TosaUtilOps.td

```diff
@@ -140,11 +141,11 @@ def Tosa_VariableReadOp : Tosa_Op<"variable.read", []> {
   let summary = "read_buffer operator";
 
   let description = [{
-    Reads the value from a pseudo-buffer resource holding a mutable tensor.
+    Reads the value from a pseudo-buffer resource holding a persistent mutable tensor.
   }];
 
   let arguments = (ins
-    SymbolNameAttr:$name
+    I32Attr:$uid
   );
 
   let results = (outs
```

Comment:
(github wouldn't let me leave a comment on the line below)

Should this be `TosaTensor: $output1` in accordance with the spec?

Reply:
same reason above.

Reply:
Updated to align with spec.